lib/render.js

```diff
@@ -79,7 +83,11 @@ var render = module.exports = function (html, options) {
   if (options.highlightSyntax) parser.use(codeWrap)
   if (options.serveImagesWithCDN) parser.use(cdnImages, {package: options.package})
 
-  return githubLinkify(parser).render(html)
+  return githubLinkify(parser)
+}
+
+render.renderPackageDescription = function (description) {
```

Comment:
what does `PackageDescription` mean here? feel like we may soon be removing this when we get rid of `packagize.js` with #282 

otherwise this is a misleading name. shouldn't affect this PR, but we should change that if it means the full `README`